axelrod/strategies/titfortat.py

```diff
@@ -300,7 +300,7 @@ def strategy(self, opponent: Player) -> Action:
             return C
         # TFT on round 2
         if len(self.history) == 1:
-            return D if opponent.history[-1:] == [D] else C
```

Comment:
No I prefer this too. This is how TFT __was__ implemented a while ago but no longer :)